url/src/host.rs

```diff
@@ -80,15 +79,34 @@ impl Host<String> {
     ///
     /// <https://url.spec.whatwg.org/#host-parsing>
     pub fn parse(input: &str) -> Result<Self, ParseError> {
```

Comment:
I didn't  add, change, or remove from the public API, but please verify this as well in case I missed something.